src/utils/ChartOptions/GrowthChartAnnotations.tsx

```diff
@@ -17,7 +17,7 @@ export const GrowthChartAnnotations = (
     };
 
     if (datasetMetadata.xAxisLabel === TimeUnitCodes.weeks) {
-        timeUnitConfig = timeUnitData.Months;
+        return [];
     }
```

Comment:
Could you remove the entire if statement, as it dosnt return anything?
```suggestion
```

Reply:
I believe the easiest way is to return []. The code will run either way, so it is either returning no annotations (return []) inside of the function like now. This way, the rest of the function will not run. 

The second alternative is to no run the function at all, but then we need to configure the code in GrowthChartBuilder. Me personally like the first alternative (return []) the most.